pkgs/by-name/ba/bash-language-server/package.nix

```diff
@@ -53,13 +53,15 @@ stdenv.mkDerivation (finalAttrs: {
 
     # Create the executable, based upon what happens in npmHooks.npmInstallHook
     makeWrapper ${lib.getExe nodejs} $out/bin/bash-language-server \
-      --prefix PATH : ${lib.makeBinPath [ shellcheck ]} \
+      --suffix PATH : ${lib.makeBinPath [ shellcheck ]} \
```

Comment:
Is this a thing expected or recommended?

Reply:
Understood.

Reply:
> I think it’s reasonable that if the user has shellcheck in their $PATH, that one should take precedence.

That's indeed a good reason to use `--suffix`. Could you explain that in the commit message please?